lib/system/alloc.nim

```diff
@@ -103,7 +128,7 @@ type
 
   MemRegion = object
     minLargeObj, maxLargeObj: int
-    freeSmallChunks: array[0..SmallChunkSize div MemAlign-1, PSmallChunk]
+    freeSmallChunks: array[0..SmallChunkSize div MinAlignment-1, array[Alignments, PSmallChunk]]
```

Comment:
Oh my... yet another array index operation in the critical path just for alignment support? Isn't there a better way?

Reply:
replaced with one array